### DISCUSSION TITLE
[Bug]: no documentation for storybook add-on in angular

### DISCUSSION THREAD

**Discussion by neerajabandi** [storybookjs/storybook#25797] [General] [answered]
### Describe the bug

I am trying to create  custom add-on in the angular app but I don't find anything in the angular. All I could see in the react and also  add-on kit is in the react. on the documentation I see angular, when I click it , I see the code in the react. I dont know how to proceed to create my custom add-on. I am working on angular 15. can someone help me with this ASAP

### To Reproduce

https://storybook.js.org/docs/addons/writing-addons/#ui-based-addons

**Comment by valentinpalkovic** [comment, marked as answer]
Storybook's Manager is written in React. If you want to create a custom addon, it is sometimes necessary to fall back or use React APIs or components to interact with the manager or register new components, which must be written in React. Can you elaborate on what exactly you would like to achieve?

**Reply by neerajabandi** [reply]
I am developing a custom add-on designed to minify JSON. Upon pasting my JSON into the add-on and clicking the 'minify' button, it will output the minified version of the JSON. I am trying to achieve this.

**Comment by neerajabandi** [comment]
I am developing a custom add-on designed to minify JSON. Upon pasting my JSON into the add-on and clicking the 'minify' button, it will output the minified version of the JSON. I am trying to achieve this.

**Comment by neerajabandi** [comment]
I found solution
